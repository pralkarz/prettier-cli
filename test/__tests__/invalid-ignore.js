import { runCli } from "../utils";

describe("ignores .prettierignore folder", () => {
  runCli("invalid-ignore", [
    "something.js",
  ]).test({
    status: 0
  });
});
