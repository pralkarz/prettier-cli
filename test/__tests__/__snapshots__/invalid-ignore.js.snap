// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ignores .prettierignore folder (stderr) 1`] = `""`;

exports[`ignores .prettierignore folder (stdout) 1`] = `""`;

exports[`ignores .prettierignore folder (write) 1`] = `[]`;
